Vic2ToHoI4/Source/Mappers/CountryMapperBuilder.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COUNTRY_MAPPER_BUILDER_H
+#define COUNTRY_MAPPER_BUILDER_H
+
+
+
+#include "CountryMapping.h"
+#include <memory>
+
+
+
+class CountryMapper::Builder
+{
+  public:
+	Builder() { countryMapper = std::make_unique<CountryMapper>(); }
+	std::unique_ptr<CountryMapper> Build() { return std::move(countryMapper); }
+	Builder& addMapping(const std::string& Vic2Tag, const std::string& HoI4Tag)
+	{
+		countryMapper->V2TagToHoI4TagMap[Vic2Tag] = HoI4Tag;
```

Comment:
Why are there no double-keyed maps in C++ outside multi_index_container that boost provides? Damn, that would help us.